src/traces/scattergl/index.js

```diff
@@ -970,13 +20,13 @@ module.exports = {
     supplyDefaults: require('./defaults'),
     crossTraceDefaults: require('../scatter/cross_trace_defaults'),
     colorbar: require('../scatter/marker_colorbar'),
-    calc: calc,
-    plot: plot,
-    hoverPoints: hoverPoints,
-    selectPoints: selectPoints,
+    calc: require('./calc'),
+    plot: require('./plot'),
+    hoverPoints: hover.hoverPoints,
+    selectPoints: require('./select'),
 
-    sceneUpdate: sceneUpdate,
-    calcHover: calcHover,
+    sceneUpdate: require('./scene_update'),
+    calcHover: hover.calcHover,
```

Comment:
No need to export these off `ScatterGl` anymore (I think).

Reply:
Good point!
Done in bb12f68.